src/modules/map/components/olMap/handler/draw/OlDrawHandler.js

```diff
@@ -695,7 +694,16 @@ export class OlDrawHandler extends OlLayerHandler {
 	}
 
 	_setSelection(ids = []) {
-		if (this._select) {
+		// Todo: abstract check for array equality (flat array) could be extracted to utils
+		const arrayEquals = (a, b) => {
+			return Array.isArray(a) &&
+				Array.isArray(b) &&
+				a.length === b.length &&
+				a.every((val, index) => val === b[index]);
```

Comment:
You can use `equals` from storeUtils.js for now. Later we can move this function to utils.js

Reply:
thx